Weather/.env

```diff
@@ -1 +1 @@
-
+# WEATHERSTACK_KEY = "6fd6bf9dd9760cacf7dbd8c105714760"
```

Comment:
We don't want this in the repository. Remove value here (or whole line). I'll create an issue to get this fixed.